x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/create_connector_flyout/header.tsx

```diff
@@ -47,6 +51,25 @@ const FlyoutHeaderComponent: React.FC<{
               <EuiText size="s" color="subdued">
                 {actionTypeMessage}
               </EuiText>
+              {featureIds && featureIds.length > 0 && (
```

Comment:
**question:** should we do the same for the edit flyout?

Reply:
I can add the type to the header title, like `Edit Email connector`. WDYT?